frontend/src/components/api/gameChanger-service-api.js

```diff
@@ -301,7 +301,8 @@ export default class GameChangerAPI {
 		const s3Bucket = cloneData?.s3_bucket ?? 'advana-raw-zone/bronze';
 		// const s3Bucket = 'advana-raw-zone';
 		const url = endpoints.thumbnailStorageDownloadPOST;
-		return axiosPOST(this.axios, url, {filenames, folder, clone_name: cloneData.clone_name, dest: s3Bucket}, {timeout: 30000})
+		const cloneName = "gamechanger"
```

Comment:
this needs to be done in the PR that switches homepage (cloneName) to gamechanger and not gamechanger-test otherwise the test page doesnt work for loading images.